octoprint_mrbeam/translations/es/LC_MESSAGES/messages.po

```diff
@@ -3999,207 +4762,199 @@ msgstr "4 marcadores están dentro del rectángulo verde"
 msgid "No black pixels are in the green rectangle"
 msgstr "No hay píxeles negros en el rectángulo verde"
 
+#: octoprint_mrbeam/templates/modals/temperature_warning.jinja2:3
+msgid "Temperature Warning, please check your Mr Beam!"
+msgstr "¡Temperatura alta, comprueba tu Mr Beam!"
+
+#: octoprint_mrbeam/templates/modals/temperature_warning.jinja2:8
+msgid ""
+"Attention:<br>The temperature in your Mr Beam is higher than usual."
+"<br><br>This is an indication of a fire. Check the working area to make sure "
+"that everything is alright!"
+msgstr ""
+"Atención:<br>La temperatura en tu Mr Beam es más alta de lo habitual."
+"<br><br>Esto es un indicador de fuego. ¡Comprueba que todo está bien en el "
+"área de trabajo!"
+
+#: octoprint_mrbeam/templates/modals/temperature_warning.jinja2:13
+msgid ""
+"Your Mr Beam can only be used again when the temperature drops further. This "
+"may take a few minutes."
+msgstr ""
+"Tu Mr Beam sólo se podrá volver a utilizar cuando la temperatura haya "
+"bajado. Esto puede tardar unos minutos."
+
+#: octoprint_mrbeam/templates/modals/temperature_warning.jinja2:15
+msgid "All good!"
+msgstr "¡Todo bien!"
+
+#: octoprint_mrbeam/templates/settings/about_settings.jinja2:5
+msgid "https://mr-beam.org/en/downloads/"
+msgstr ""
```

Comment:
This is also empty

Reply:
@irlaec

Reply:
We don't have a Spanish link, so all the links are left untranslated, then it is easier to find which ones are missing if we ever have a Spanish link

Reply:
But wouldn't t be better to link then to the english instead of don't have a working link?

Reply:
It will already show the English link. If there is no translation provided, the default is shown

Reply:
But the translation provided is "" so it should show "" instead, did you test it?

Reply:
Confirmed, it is tested and the English link is shown